app/src/app/home/home.component.sass

```diff
@@ -18,8 +19,11 @@ div#logos
   margin-top: 2em
   justify-content: center
   align-items: center
+  img
+    height: auto
+    width: 96%
```

Comment:
Do you have a reason for not setting the width to 100%? If not please use 100% instead. I strongly believe it wont change anything, except preventing readers to wonder why it's not 100%.